File: py/testPerson.py
from Person import Person

if __name__=="__main__":
	psn=Person("Taro", "Tanaka", 25, 175.5)
	print(psn.firstName)
	print(psn.lastName)
	print(psn.age)
	print(psn.height)
	print(psn.fullName())
	print(psn.fullName(True))
	print(psn.fullName(True, ","))
	psn.viewStatus()

File: ts/testPerson.ts
"use strict";
import {Person} from "./Person";

(function main(){
	const psn=new Person("Taro", "Tanaka", 25, 175.5);
	console.log(psn.firstName);
	console.log(psn.lastName);
	console.log(psn.age);
	console.log(psn.height);
	console.log(psn.fullName());
	console.log(psn.fullName(true));
	console.log(psn.fullName(true, ","));
	psn.viewStatus();
})();
